components/legacy_team_sidebar/legacy_team_sidebar_controller.tsx

```diff
@@ -185,22 +196,22 @@ export default class LegacyTeamSidebar extends React.PureComponent {
             destinationIndex,
             result.draggableId,
         );
-        updateTeamsOrderForUser(newTeamsOrder.map((o) => o.id));
+        updateTeamsOrderForUser(newTeamsOrder.map((o: Team) => o.id));
         this.setState({teamsOrder: newTeamsOrder});
     }
 
     render() {
-        const root = document.querySelector('#root');
+        const root: Element | null = document.querySelector('#root');
         if (this.props.myTeams.length <= 1) {
-            root.classList.remove('multi-teams');
+            (root as Element).classList.remove('multi-teams');
```

Comment:
probably better written as `root!` and not `(root as Element)`

Reply:
Fixed in the additional commit